cmds/core/dhclient/dhclient.go

```diff
@@ -43,22 +43,9 @@ func main() {
 		ifName = flag.Args()[0]
 	}
 
-	ifRE := regexp.MustCompilePOSIX(ifName)
-
-	ifnames, err := netlink.LinkList()
+	filteredIfs, err := util.Interfaces(ifName)
```

Comment:
I think pxeboot does the same thing, too. use Interfaces there as well

Reply:
will do. I figured this had to be more than one place by now :-)

Reply:
done